addons/main/script_debug.hpp

```diff
@@ -43,7 +43,7 @@ PERFORMANCE COUNTERS SECTION
 // To Use: [] call ace_common_fnc_dumpPerformanceCounters;
 
 #ifdef ENABLE_PERFORMANCE_COUNTERS
-    #define CBA_fnc_addPerFrameHandler { _ret = [(_this select 0), (_this select 1), (_this select 2), #function] call CBA_fnc_addPerFrameHandler; if(isNil "ACE_PFH_COUNTER" ) then { ACE_PFH_COUNTER=[]; }; ACE_PFH_COUNTER pushBack [[_ret, __FILE__, __LINE__], [(_this select 0), (_this select 1), (_this select 2)]];  _ret }
+    #define CBA_fnc_addPerFrameHandler { private _ret = [(_this select 0), (_this select 1), (_this select 2)] call CBA_fnc_addPerFrameHandler; if(isNil "ACE_PFH_COUNTER" ) then { ACE_PFH_COUNTER=[]; }; ACE_PFH_COUNTER pushBack [[_ret, __FILE__, __LINE__], [(_this select 0), (_this select 1), (_this select 2)]];  _ret }
```

Comment:
I wonder if this is faster, it's at least more readable:
```suggestion
    #define CBA_fnc_addPerFrameHandler { private _ret = (_this select [0, 2]) call CBA_fnc_addPerFrameHandler; if(isNil "ACE_PFH_COUNTER" ) then { ACE_PFH_COUNTER=[]; }; ACE_PFH_COUNTER pushBack [[_ret, __FILE__, __LINE__], (_this select [0, 2])];  _ret }
```
The changed parts don't need the parenthesis, though, but I'm sure they'll be wanted regardless.

And yes, I know this is a debugging code.

Reply:
as far as I can tell it could just be `_this` as we shouldn't have a 4th arg
but this is fine and keeps the original meaning